tests/test_lint.py

```diff
@@ -30,7 +30,7 @@ class TestLint(unittest.TestCase):
     def test_critical_missingfiles_example(self):
         """Tests for missing nextflow config and main.nf files"""
         lint_obj = nf_core.lint.PipelineLint(PATH_CRITICAL_EXAMPLE)
-        lint_obj.check_files_exist()
+        lint_obj.lint_pipeline()
```

Comment:
How come switching back to this main command? This one runs all tests, rather than just the one that this test is testing...

Reply:
A damn, this was a mistake, when I resolved a local merge conflict. I will fix this.